content/docs/setup/upgrade-notice/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 1.1 Upgrade Notice
+description: Breaking or surprising changes in Istio 1.1.
```

Comment:
stating this as "breaking" changes is kind of bad, given that 1.0 is supposed to be a stable release.. I would state something like "behavioral changes" or something else?

Reply:
I personally don't care because I think we are just being transparent, but I will say that this is the third complaint I've heard about using the phrase 'breaking change' so it will probably be an issue for many people

Reply:
Word games, right? Behavioral changes is a euphemism for "breaking" IMO. How about:

"Important changes operators must understand before upgrading".